StudyHub/StudyHub.BLL/Services/Auth/PasswordService.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Microsoft.AspNetCore.Identity;
+using StudyHub.BLL.Services.Interfaces.Auth;
+using StudyHub.Common.DTO.AuthDTO;
+using StudyHub.Common.Exceptions;
+using StudyHub.Entities;
+
+namespace StudyHub.BLL.Services.Auth;
+public class PasswordService : IPasswordService
+{
+    private readonly UserManager<User> _userManager;
+    public PasswordService(UserManager<User> userManager)
+    {
+        _userManager = userManager;
+    }
+    public async Task<string> ForgotPassword(ForgotPasswordDTO dto)
+    {
+        var user = await _userManager.FindByEmailAsync(dto.Email);
+
+        if (user == null)
```

Comment:
I think its better to make like ,,?? throw new ..... ,,

Reply:
why?

Reply:
> why?

I think this way ForgotPassword (and add suffix **Async** to it btw) will look much simpler or at least shorter, idk